setup.py

```diff
@@ -1,36 +1,46 @@
-import os
-
 from setuptools import setup
 
-DIR = os.path.dirname(os.path.abspath(__file__))
-
 extras = {
     "docs": [
         "mkdocs-include-markdown-plugin==2.8.0",
         "mkdocs==1.0.4",
         "pymdown-extensions",
-        f"mkdocs_execute_python_plugin @ file://{DIR}/docs/mkdocs_execute_python_plugin",
+        "mkdocs-macros-plugin",
     ],
     "test": [
         "jinja2",
-        "pytest>=4",
+        "pytest>=6",
```

Comment:
Oops, I tried to grab the latest setup.py with `git checkout main -- setup.py`, forget it had changed. The extra changes where will go away if we rebase.

Reply:
Fixed before pushing the reduced indirection commit.